lib/Ohmbrewer_Rhizome.h

```diff
@@ -247,6 +253,17 @@ namespace Ohmbrewer {
          * @return Error or success code, according to the requirements specified by addSprout
          */
         int addRIMS(char* params);
+
+        /**
+         * Adds a sprout to the list and rebuids the index
+         * @param equipment The sprout being added
+         */
+        void saveNewSprout(Equipment* sprout);
```

Comment:
Based on our conversation earlier this week (and that it can really go either way and the semester is almost over), let's just go with the name as you have it if you haven't already made the change. Renaming later isn't hard if it really matters.

I think you're ready to squash and merge this.